gecoscc-installer.sh

```diff
@@ -223,7 +218,36 @@ install_template "/opt/gecosccui-$GECOSCC_VERSION/supervisord.conf" supervisord.
 mkdir -p /opt/gecosccui-$GECOSCC_VERSION/supervisor/run
 mkdir -p /opt/gecosccui-$GECOSCC_VERSION/supervisor/log
 chkconfig supervisord on
+[ ! `id -u gecoscc 2> /dev/null` ] && \
+ adduser -d /opt/gecosccui-$GECOSCC_VERSION \
+ -r \
+ -s /bin/false \
+ gecoscc
+[ ! -d /opt/gecosccui-$GECOSCC_VERSION/sessions ] && \
+ mkdir -p /opt/gecosccui-$GECOSCC_VERSION/sessions
+[ ! -d /opt/gecoscc/media/users ] && \
+ mkdir -p /opt/gecoscc/media/users
+chown -R gecoscc:gecoscc /opt/gecoscc
+chown -R gecoscc:gecoscc /opt/gecosccui-$GECOSCC_VERSION/sessions/
+chown -R gecoscc:gecoscc /opt/gecosccui-$GECOSCC_VERSION/supervisor/
+chown -R gecoscc:gecoscc /opt/gecosccui-$GECOSCC_VERSION/supervisord.conf
+pip install --upgrade gevent==1.2.1
```

Comment:
because `gecoscc-ui` requirements points to `gevent==1.0` and reverse the first installation. Plus, `gevent` must be updated to that version before leaving installation process.